samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages_es.properties

```diff
@@ -264,3 +264,5 @@ transfer_pool_select_all=Seleccione todas las bater\u00edas
 
 ## S2U-25 Question pools export
 t_exportPool=Exportar
+
+datatables_entities=elementos
```

Comment:
Maybe could we add before the property the corresponding comment

## S2U-18 ...

Reply:
Yes please, 

Add ## S2U-18 to all the files.